packages/shared/src/interfaces/customLLMProviderConfigSchemas.ts

```diff
@@ -0,0 +1,13 @@
+import { z } from "zod";
+
+export const BedrockConfigSchema = z.object({ region: z.string() });
+export type BedrockConfig = z.infer<typeof BedrockConfigSchema>;
+
+export const BedrockCredentialSchema = z
```

Comment:
Do we have the option to update this? I assume these values will be invalid after 1h.

Reply:
Only region is put into config, accesskeyId and key are stringified and encrypted as the secretkey